stripe/src/main/java/com/stripe/android/view/AddPaymentMethodActivityStarter.java

```diff
@@ -175,6 +182,12 @@ Builder setPaymentConfiguration(@Nullable PaymentConfiguration paymentConfigurat
                 return this;
             }
 
+            @NonNull
+            Builder setAddPaymentMethodFooter(@LayoutRes int addPaymentMethodFooter) {
```

Comment:
let's make this public, because a user may want to launch `AddPaymentMethodActivity` directly

Reply:
👍